experimental/examples/bubblebench/editable-shared-tree/webpack.config.js

```diff
@@ -0,0 +1,69 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const fluidRoute = require("@fluid-tools/webpack-fluid-loader");
+const path = require("path");
+const { merge } = require("webpack-merge");
+const webpack = require('webpack'); //to access built-in plugins
+
+module.exports = (env) => {
+    const isProduction = env?.production;
+
+    return merge(
+        {
+            entry: {
+                main: "./src/index.ts",
+            },
+            resolve: {
+                extensions: [".ts", ".tsx", ".js"],
+            },
+            module: {
+                rules: [
+                    {
+                        test: /\.tsx?$/,
+                        loader: "ts-loader",
+                    },
+                ],
+            },
+            output: {
+                filename: "[name].bundle.js",
+                path: path.resolve(__dirname, "dist"),
+                library: "[name]",
+                // https://github.com/webpack/webpack/issues/5767
+                // https://github.com/webpack/webpack/issues/7939
+                devtoolNamespace: "fluid-example/bubblebench",
+                libraryTarget: "umd",
+            },
+            // This impacts which files are watched by the dev server (and likely by webpack if watch is true).
+            // This should be configurable under devServer.static.watch
+            // (see https://github.com/webpack/webpack-dev-server/blob/master/migration-v4.md) but that does not seem to work.
+            // The CLI options for disabling watching don't seem to work either, so this may be a symptom of using webpack4 with the newer webpack-cli and webpack-dev-server.
+            watchOptions: {
+                ignored: "**/node_modules/**",
+            },
+            plugins: [
+                // Uncaught reference error: process is not defined at
+                // ../../../../packages/dds/tree/lib/feature-libraries/editable-tree/editableTree.js
```

Comment:
If we have a bug where editable tree needs a process polyfill, this does not the like the place to document that.

Reply:
Removed in commit # c7f0c5